utils/rollup-build-target.mjs

```diff
@@ -104,7 +105,7 @@ function buildTarget(buildType, moduleFormat, input = 'src/index.js', buildDir =
 
     const outputExtension = {
         es5: '.js',
-        es6: '.mjs'
+        es6: shouldBundle ? '.esm.js' : '.mjs'
```

Comment:
In the current file extension scheme `.js` means UMD, shouldn't we keep `.mjs`? Maybe something like `playcanvas.bundled.mjs`